_posts/2022-05-02-the-na-bug/the-na-bug.Rmd

```diff
@@ -0,0 +1,111 @@
+---
+title: "Ooh na na... where are my sodium labs?"
+description: |
+  The NA bug, or, what happens when the same word is used in different contexts.
+author:
+  - name: Chloe Pou-Prom
+    affiliation: DSAA, Unity Health Toronto
+    affiliation_url: https://chartdatascience.ca
+date: 2022-05-02
+output:
+  distill::distill_article:
+    self_contained: false
+categories:
+  - language-R
+  - project-chartwatch
+  - post-miscellaneous
+draft: true
```

Comment:
I had to toggle this to `false` for the markdown to render nicely (some of the styling looked a bit weird). Might be useful to include this info in the README or in the guidelines/style clickup document.